lib/m2ts/caption-stream.js

```diff
@@ -1427,7 +1437,11 @@ var Cea608Stream = function(field, dataChannel) {
         // increments the column cursor by 4, so we can get the desired
         // column position by bit-shifting to the right (to get n/2)
         // and multiplying by 4.
-        this.column_ = ((data & 0xe) >> 1) * 4;
+        const indentations = ((data & 0xe) >> 1);
+
+        this.column_ = indentations * 4;
+        // add to the number of indentations for positioning
+        this.nonDisplayed_[this.row_].indent += indentations;
```

Comment:
Do we need to add the indent to the `this.displayed_` buffer here as well?

Reply:
Good question, and I don't believe so. I think the `this.nonDisplayed_` buffer is basically where the caption data is manipulated until the end of a caption. At that instance, [we swap everything over to `this.displayed_`](https://github.com/videojs/mux.js/blob/dc56f1b79224bf20dc34ec1e949df6b18c2ffde4/lib/m2ts/caption-stream.js#L1290).